app/v1/services/EnrolmentsAuthService.scala

```diff
@@ -43,27 +44,36 @@ class EnrolmentsAuthService @Inject()(val connector: AuthConnector) {
     .flatMap(_.getIdentifier("AgentReferenceNumber"))
     .map(_.value)
 
+  def buildPredicate(predicate: Predicate): Predicate =
+    predicate and {
+      if (appConfig.confidenceLevelConfig.authValidationEnabled) {
+        (Individual and ConfidenceLevel.L200) or Organisation or Agent
+      } else {
+        Individual or Organisation or Agent
```

Comment:
Could this just be `EmptyPredicate` or this method be reworded to remove this line? I don't know if this is necessary.